Sources/hessian.f90

```diff
@@ -56,7 +56,7 @@ MODULE hessian
                            mupar, mupar0, maxeigen, mindiag
       LOGICAL, PUBLIC   :: l_Compute_Hessian, l_Diagonal_Only
       LOGICAL, PUBLIC, PARAMETER :: l_Hess_sym=.FALSE.                     !<forces symmetrization of Hessian if TRUE (only implemented for Thomas, 1 proc)
-      INTEGER, PUBLIC :: HESSPASS=0
```

Comment:
For new code we should add doxygen documentation for the variable. It should be put on its own line with a Doxygen comment. Also consider a more descriptive name for the counter_g variable.
 
```
      INTEGER, PUBLIC :: HESSPASS=0
!>  Counter for the number of Levenberg-Marquart parameter resets.
      INTEGER, PUBLIC :: counter_g=0
```